preview-maker.js

```diff
@@ -7,13 +7,18 @@ const sizeOf = require(`image-size`);
 const { serializeToString: serialize } = new XMLSerializer();
 const Color = require(`@snejugal/color`);
 const rgbToHsl = Color.rgbToHsl;
+const puppeteer = require("puppeteer");
 
 const RENDER_CONFIG = {
   density: 150,
 };
-
 const parser = new DOMParser();
 
+let browser = null;
+const newBrowser = async () => {
+  browser = await puppeteer.launch();
+};
```

Comment:
```suggestion
const browser = puppeteer.launch();
```

then just `await browser` where needed, see next suggestion